src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaManagerDataStore.cs

```diff
@@ -38,10 +38,14 @@ public async Task ExecuteScriptAndCompleteSchemaVersionAsync(string script, int
 
             try
             {
-                var server = new Server(serverConnection);
+                // FullSchemaSnapshot script Inserts into SchemaVersion table with stated status
```

Comment:
Can we reword this to FullSchemaSnapshot script(x.sql) inserts 'started' status into the SchemaVersion table itself.